src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbByTable.java

```diff
@@ -269,9 +281,6 @@ private void setDefaults() {
             if (delimiter == null) {
                 delimiter = ".";
```

Comment:
why not just set these values (`delimiter` (as `"."`) and `billingMode` (as `BillingMode.PROVISIONED`)) where the fields are defined (and get rid of the `setDefaults` method)?

Reply:
I think we're just following the pattern of setting everything in `setDefaults()`.  Although, admittedly, this code style violation forced me away from this.